### PR TITLE
updated README

### PR DESCRIPTION
**Pull request by shrmn**
Creating pull request per instructions.

### CODE REVIEW

**Comment by shrmn** [comment]
Finished MVP. Working on stretch now.

**Comment by shrmn** [comment]
Completed stretch exercises.

File: challenges/classes.js [hunk 1]
```diff
@@ -1,7 +1,56 @@
...
+
+// Stretch Task: Extend the base class CuboidMaker with a sub class called CubeMaker.  Find out the formulas for volume and surface area for cubes and create those methods using the dimension properties from CuboidMaker.  Test your work by logging out your volume and surface area.
+
+// Volume of the cube = l × l × l = l³
+// Surface of the cute = 6 * (a*a)
+
+class CubeMaker extends CuboidMaker {
+  constructor(cubeAttrs) {
+    super(cubeAttrs);
+    // this.length = cubeAttrs.length;
```
Comment (BrandonVilaychith) [inline challenges/classes.js:37]: Unneeded since CubeMaker extends CuboidMaker

File: challenges/functions.js [hunk 2]
```diff
@@ -7,25 +7,43 @@
...
 
+function add(a, b){
+  return a+b;
+};
+
+function multiply(a, b){
+  return a*b;
+};
+
+function greeting(a, b){
```
Comment (BrandonVilaychith) [inline challenges/functions.js:29]: I would use more descriptive names for the parameters

File: challenges/objects-arrays.js [hunk 3]
```diff
@@ -46,26 +70,39 @@ const graduates = [{"id":1,"first_name":"Cynde","university":"Missouri Southern
...
 
-/* Request 1: Create a new array called universities that contains all the univeristies in the graduates array.  
+// Request 1: Create a new array called universities that contains all the univeristies in the graduates array.  
 
-Once you have the new array created, sort the universities alphabetically and log the result. */
-const universities = [];
-console.log(universities)
+// Once you have the new array created, sort the universities alphabetically and log the result.
+
+const universities = graduates.map((student) => student.university).sort();
```
Comment (BrandonVilaychith) [inline challenges/objects-arrays.js:77]: We fixed this during our meeting but .sort can be chained on the array that comes from the .map

**Pull request merged**